cmake/macros/macro_deal_ii_setup_target.cmake

```diff
@@ -103,9 +103,8 @@ MACRO(DEAL_II_SETUP_TARGET _target)
     SET(_build "RELEASE")
   ENDIF()
 
-  SET_PROPERTY(TARGET ${_target} APPEND PROPERTY
-    INCLUDE_DIRECTORIES "${DEAL_II_INCLUDE_DIRS}"
-    )
+  TARGET_INCLUDE_DIRECTORIES(${_target} SYSTEM PRIVATE ${DEAL_II_INCLUDE_DIRS})
```

Comment:
We have to check when `TARGET_INCLUDE_DIRECTORIES` was introduced.

Reply:
according to https://stackoverflow.com/questions/20643368/cmake-different-include-directories-for-different-targets/20643487#20643487 -- 2.8.12